raven-android/src/main/java/com/getsentry/raven/android/event/helper/AndroidEventBuilderHelper.java

```diff
@@ -0,0 +1,40 @@
+package com.getsentry.raven.android.event.helper;
+
+import android.content.Context;
+import android.content.pm.PackageManager;
+import android.util.Log;
+import com.getsentry.raven.event.EventBuilder;
+import com.getsentry.raven.event.helper.EventBuilderHelper;
+
+/**
+ * EventBuilderHelper that makes use of Android Context to populate some Event fields.
+ */
+public class AndroidEventBuilderHelper implements EventBuilderHelper {
+
+    /**
+     * Logger tag.
+     */
+    public static final String TAG = AndroidEventBuilderHelper.class.getName();
+
+    private Context ctx;
+
+    /**
+     * Construct given the provided Android {@link Context}.
+     *
+     * @param ctx Android application context.
+     */
+    public AndroidEventBuilderHelper(Context ctx) {
+        this.ctx = ctx;
+    }
+
+    @Override
+    public void helpBuildingEvent(EventBuilder eventBuilder) {
+        try {
+            int versionCode = ctx.getPackageManager().getPackageInfo(ctx.getPackageName(), 0).versionCode;
+            eventBuilder.withRelease(Integer.toString(versionCode));
```

Comment:
This was from that `Sentry-Raven` PR. Tested and works. Figure we'll add a lot more here after the original Android code lands.

One odd thing here though (and I didn't see an obvious way out considering what is provided to this method) -- we *always* override the release. What if I user wanted to set it to a string that had more meaning to them? On the other hand, this Android `versioncode` is a specific and well known thing. We could just force it upon people.